### PR TITLE
Pin BLAS implementation to OpenBLAS

### LINKED ISSUE #199: warnings in sys_info() output
This showed up in a user's `mne.sys_info()` output [on the forum](https://mne.discourse.group/t/valueerror-all-picks-must-be-n-channels-73-got-73/7375/35).
```
├☑ pyvista           0.41.1C:\ProgramData\mne-python\1.5.0_0\Lib\site-packages\threadpoolctl.py:1010: RuntimeWarning: 
Found Intel OpenMP ('libiomp') and LLVM OpenMP ('libomp') loaded at
the same time. Both libraries are known to be incompatible and this
can cause random crashes or deadlocks on Linux when loaded in the
same Python program.
Using threadpoolctl may cause crashes or deadlocks. For more
information and possible workarounds, please see
    https://github.com/joblib/threadpoolctl/blob/master/multiple_openmp.md

  warnings.warn(msg, RuntimeWarning)
 (OpenGL 4.5.0 NVIDIA 536.23 via NVIDIA GeForce RTX 2080 SUPER/PCIe/SSE2)
├☑ pyvistaqt         0.0.0
```

Seems worth digging into how this happened, and trying to prevent it.

### PR DESCRIPTION
**Pull request by hoechenberger**
Closes #199

### CODE REVIEW

File: assets/post_install_linux.sh [hunk 1]
```diff
@@ -21,5 +21,8 @@ ${PREFIX}/bin/conda env config vars set PYTHONNOUSERSITE=1
 echo "ℹ️ Disabling mamba package manager banner."
 ${PREFIX}/bin/conda env config vars set MAMBA_NO_BANNER=1
 
+echo "ℹ️ Pinning BLAS implementation to OpenBLAS"
+echo "libblas=*=*openblas" >>${PREFIX}/conda-meta/pinned
```
Comment (hoechenberger) [inline assets/post_install_linux.sh:25]: This prevents accidental switching of the BLAS implementation later on by the user when installing packages manually

Comment (larsoner) [reply assets/post_install_linux.sh:25]: Nice!

File: recipes/mne-python_1.5/construct.yaml [hunk 2]
```diff
@@ -82,12 +82,14 @@ specs:
   - pyriemann =0.5
   - pyprep =0.4.2
   - openmeeg =2.5.6
+  # BLAS
+  # - openblas =0.3.23
```
Comment (larsoner) [inline recipes/mne-python_1.5/construct.yaml:86]: I think I'd rather keep this, it should help speed up resolution, and we can rely on `outdated` to tell us to update
```suggestion
  - openblas =0.3.23
```

**Review by larsoner — COMMENTED** [review]
Looks like it's just `outdated` that needs fixing

**Comment by hoechenberger** [comment]
> Looks like it's just `outdated` that needs fixing

Please feel free to push the required changes, I'll be busy until Friday I suppose

File: recipes/mne-python_1.5/construct.yaml (hunk 2 above)
Comment (hoechenberger) [reply recipes/mne-python_1.5/construct.yaml:86]: Fair enough!

**Comment by larsoner** [comment]
Deleted the branch from `upstream` will open PR from my fork

**Pull request closed without merging**